docarray/array/storage/sqlite/getsetdel.py

```diff
@@ -52,20 +52,28 @@ def _del_docs_by_ids(self, ids: str) -> Iterable['Document']:
         self._commit()
 
     def _load_offset2ids(self):
-        r = self._sql(
-            f"SELECT doc_id FROM {self._table_name} ORDER BY item_order",
-        )
-        self._offset2ids = Offset2ID(list(map(itemgetter(0), r)))
+        if self._list_like:
+            r = self._sql(
+                f"SELECT doc_id FROM {self._table_name} ORDER BY item_order",
+            )
+            self._offset2ids = Offset2ID(
+                list(map(itemgetter(0), r)), list_like=self._list_like
+            )
+            return self._offset2ids
+        else:
+            self._offset2ids = Offset2ID([], list_like=self._list_like)
```

Comment:
I think we should set it to None

Reply:
None will cause errors, like code below:
<img width="490" alt="image" src="https://user-images.githubusercontent.com/8253226/201035603-64c47ea4-4c31-4ea7-b522-3360b69dbfc6.png">
because we need to disable the functionality of the Offset2id to make it a shell, if we make it None, then some of the code will break and it is contradictory to our design purpose.

Reply:
okey perfect